### ISSUE TITLE
Maybe inconsistency regarding "UC Browser"

### ISSUE THREAD

**Issue by hgoebl** [serbanghita/Mobile-Detect#451]
I could be wrong, but maybe the handling of "UC Browser" is not consistent:

```
protected static $browsers = array(
    ...
    'UCBrowser'       => 'UC.*Browser|UCWEB',


protected static $properties = array(
    ...
    'UC Browser'    => 'UC Browser[VER]',
```

Here is an example of a current user-agent string of such browser (maybe in former times it looked different, I don't know):

> Mozilla/5.0 (Linux; U; Android 4.1.2; zh-CN; HUAWEI C8815 Build/HuaweiC8815) AppleWebKit/534.30 (KHTML, like Gecko) Version/4.0 UCBrowser/10.7.0.634 U3/0.8.0 Mobile Safari/534.30

IMHO the value for 'UC Browser' in $properties array could be changed to `UC.*Browser\/?[VER]`.

**Comment by serbanghita** [comment]
@hgoebl thanks, I've noticed this also while working on `3.0.0`. I'll make the patch in the next minor version.
I think it will be `array('UCWEB[VER]', 'UC.*Browser/[VER]')`

**Comment by serbanghita** [comment]
@hgoebl I kept `UCBrowser` as a key name for consistency. Updated the version detection a bit.

**Comment by serbanghita** [comment]
Fix available from `2.8.30`
